Framework/script/RepoCleaner/o2-qc-repo-cleaner

```diff
@@ -223,6 +229,20 @@ def storeMonitoringMetrics(success, duration):
         logging.warning(f"File {socket_file} does not exist, no monitoring metrics stored.")
 
 
+def storeCrucialLog(message):
+    """
+    Store few but very important messages to the file ~/repocleaner_logs.txt
+    :param message:
+    :return:
+    """
+    logs_filename = str(Path.home()) + "/repocleaner_logs.txt"   # very limited but important logs
+    try:
+        f = open(logs_filename, "a")
```

Comment:
Will it behave if we have to repocleaners keeping a file open at the same time? (e.g. one took long enough to be still active once a new instance starts)

Reply:
The repocleaner checks (in the current version not installed in the CC7 QCDB) that there are no other repocleaner running.